bundles/org.openhab.core.persistence/src/main/java/org/openhab/core/persistence/internal/PersistenceManagerImpl.java

```diff
@@ -158,6 +160,39 @@ private void handleStateEvent(Item item, boolean onlyChanges) {
         }
     }
 
+    /**
+     * Calls all persistence services which use change or update policy for the given item
+     *
+     * @param item the item to persist
+     * @param onlyChanges true, if it has the change strategy, false otherwise
+     */
+    private void handleHistoricStateEvent(Item item, State state, ZonedDateTime dateTime, boolean onlyChanges) {
```

Comment:
What is the `onlyChanges` parameter used for? If I don't overlook something, you always call it with `false`.